cmd/watch.go

```diff
@@ -43,6 +49,13 @@ type Event struct {
 	Type   EventType `json:"type"`
 }
 
+// Source obtains the information of the client which generated the event.
+type Source struct {
+	IP        string `json:"ip"`
+	Port      string `json:"port"`
+	UserAgent string `json:"userAgent"`
+}
```

Comment:
It will be better to make this part of `Event` struct because:

* it is part of the event
* results in simpler code as you don't need another type of
```
        struct {
		Event  Event
        	Source Source
	}
```